### ISSUE TITLE
Update character tables with new Unicode 11 characters

### ISSUE THREAD

**Issue by adrianwong** [n8willis/opentype-shaping-documents#36]
Just noticed that our character tables haven't yet been updated to include new characters introduced in Unicode 11. These should need updating:

* Arabic (Extended-A)
* Bengali
* Devanagari (Extended)
* Gurmukhi
* Hebrew
* Kannada
* Mongolian
* Nko
* Telugu

[Unicode's delta code charts](http://www.unicode.org/charts/PDF/Unicode-11.0/)

**Comment by n8willis** [comment]
Right. Will have to check against HarfBuzz's categorization and special-case handling, too, of course.

**Comment by adrianwong** [comment]
Thanks Nathan!
